lib/auth/webauthnwin/api.go

```diff
@@ -33,10 +44,211 @@ const (
 	AttachmentPlatform
 )
 
+// nativeWebauthn represents the native windows webauthn interface.
+// Implementors must provide a global variable called `native`.
```

Comment:
Interface is defined in common file. 
[Further in codebase](https://github.com/gravitational/teleport/blob/d60125c816da6e6eec74eaa78e79e57cbb208999/lib/auth/webauthnwin/api.go#L93) we use `native` variable which implements given interface, example: 
```
func Login() error {
...
resp, err := native.GetAssertion()
}
``` 
and native is added in both [windows](https://github.com/gravitational/teleport/blob/d60125c816da6e6eec74eaa78e79e57cbb208999/lib/auth/webauthnwin/webauthn_windows.go#L48) and [non_windows](https://github.com/gravitational/teleport/blob/d60125c816da6e6eec74eaa78e79e57cbb208999/lib/auth/webauthnwin/webauthn_other.go#L26) build tags.
Usage of interface allows us to mock it during UT. 


Do you have suggestions how to document it in more clear way?
```
Package variable `native` must be used to provide implementation of that interface.
For windows platform it will be real implementation and for other noop.
```

Reply:
I see now. I think it's fine as-is.